polish_example_json.py

```diff
@@ -2,148 +2,188 @@
 import random
 import string
 from functools import reduce
+from pathlib import Path
+from typing import Dict
 
+CURRENT_DRAFT_VERSION = '0.6.0-DRAFT'
```

Comment:
Highlighting this as this file contains the important changes, and it's buried in the `Files Changed` view.